tests/tests-asn1c-compiler/18a-class-OK.asn1.-B

```diff
@@ -0,0 +1,38 @@
+////////////////////// moduletestclass.proto //////////////////////
+// Protobuf generated from /18a-class-OK.asn1 by asn1c-0.9.29
```

Comment:
If you share your ideas in the mapping doc about how to implement inheritance that would be great. there are different ways we can solve the problem but perhaps we should see which one is better.